tests don't work with latest release of cmocka

With cmocka 1.1.6: 

```
[edrex@chip:~/src/github.com/alex-courtis/way-displays]$ make test
make -f tst/GNUmakefile tst-all
make[1]: Entering directory '/home/edrex/src/github.com/alex-courtis/way-displays'
gcc -pedantic -Wall -Wextra -Werror -Wno-unused-parameter -O3 -g -std=gnu17 -Wold-style-definition -Wstrict-prototypes -I/nix/store/svjch48rg6cw70b8sxckrz7a0gd4avww-wayland-1.21.0-dev/include -I/nix/store/29xdwlhjs5s6118wylbabjrc9ld185x6-yaml-cpp-0.7.0/include -I/nix/store/v6hil4fnxnn2amlxvn87ins050zbyihb-libinput-1.22.1-dev/include -I/nix/store/vsr29xlfafz0mksyaf31n515hd6cdf2v-systemd-253.1-dev/include -I/nix/store/2qa5ixdwf5f0wx2vrrf4qs8i6p3cf1ab-cmocka-1.1.6/include -Iinc -Ipro -D_GNU_SOURCE -DVERSION=\""1.7.2-SNAPSHOT"\" -DROOT_ETC=\"/etc\"  -c -o tst/tst-cfg.o tst/tst-cfg.c
In file included from tst/tst-cfg.c:2:
tst/asserts.h: In function ‘_assert_wl_fixed_t_equal_double’:
tst/asserts.h:16:17: error: implicit declaration of function ‘cmocka_print_error’; did you mean ‘cm_print_error’? [-Werror=implicit-function-declaration]
   16 |                 cmocka_print_error("%g != %g\n", wl_fixed_to_double(a), b);
      |                 ^~~~~~~~~~~~~~~~~~
      |                 cm_print_error
cc1: all warnings being treated as errors
make[1]: *** [<builtin>: tst/tst-cfg.o] Error 1
make[1]: Leaving directory '/home/edrex/src/github.com/alex-courtis/way-displays'
make: *** [GNUmakefile:77: test] Error 2
```

Some [functions are renamed](https://gitlab.com/cmocka/cmocka/-/commit/9994ceea95309906cd29602cfd5427b6c29ef57b) in the master branch, so I guess you have that. Seems like it should target the released branch, unless there's some essential fix that's unreleased?

Found while working on #86.

--- a/tst/asserts.h
+++ b/tst/asserts.h
@@ -13,7 +13,7 @@
 void _assert_wl_fixed_t_equal_double(wl_fixed_t a, double b, const char * const file, const int line) {
 
 	if (a != wl_fixed_from_double(b)) {
-		cmocka_print_error("%g != %g\n", wl_fixed_to_double(a), b);
+		cm_print_error("%g != %g\n", wl_fixed_to_double(a), b);
 		_fail(file, line);
 	}
 }
@@ -37,7 +37,7 @@ void _assert_heads_equal(struct SList *a, struct SList *b, const char * const fi
 			ep += sprintf(ep, "\n .name = '%s', .description = '%s',", head->name, head->description);
 		}
 
-		cmocka_print_error("assert_heads_equal\nactual:%s\nexpected:%s\n\n", actual, expected);
+		cm_print_error("assert_heads_equal\nactual:%s\nexpected:%s\n\n", actual, expected);
 		_fail(file, line);
 	}
 }
@@ -45,7 +45,7 @@ void _assert_heads_equal(struct SList *a, struct SList *b, const char * const fi
 
 void _assert_head_position(struct Head *head, int32_t x, int32_t y, const char * const file, const int line) {
 	if (head->desired.x != x || head->desired.y != y) {
-		cmocka_print_error("assert_head_position %s (%d, %d) != (%d, %d)\n", head->name, head->desired.x, head->desired.y, x, y);
+		cm_print_error("assert_head_position %s (%d, %d) != (%d, %d)\n", head->name, head->desired.x, head->desired.y, x, y);
 		_fail(file, line);
 	}
 }
@@ -54,7 +54,7 @@ void _assert_head_position(struct Head *head, int32_t x, int32_t y, const char *
 
 void _assert_equal_cfg(struct Cfg *a, struct Cfg *b, const char * const file, const int line) {
 	if (!cfg_equal(a, b)) {
-		cmocka_print_error("assert_cfg_equal\nactual:\n\n%s\nexpected:\n\n%s\n\n", marshal_cfg(a), marshal_cfg(b));
+		cm_print_error("assert_cfg_equal\nactual:\n\n%s\nexpected:\n\n%s\n\n", marshal_cfg(a), marshal_cfg(b));
 		_fail(file, line);
 	}
 }

